iOS/FVM/Framework/GestureHandlers/PanGestureHandler.swift

```diff
@@ -19,7 +19,7 @@ fileprivate struct NumberOfTouches {
 
 fileprivate struct RotateConstraint {
     static let maxHeightRatioXDown: Float = -0.15
-    static let maxHeightRatioXUp: Float = 0.45
+    static let maxHeightRatioXUp: Float = 0.35
```

Comment:
?? :D

Reply:
Precise question :D If you're asking about value, then it's used for driving down max angle of rotation. If about name, then yep, I shall change it to something more clear.